### PR TITLE
Get parameter application (not final)

### PR DESCRIPTION
**Pull request by orelgueta**
This partly answers #47, but it's not done (and generally not very useful, it's just there for the DB example). 
However, I need to merge this PR because of a problem I created in the environment a while back...

### CODE REVIEW

**Comment by orelgueta** [comment]
I am going to merge it, so I can keep playing with the docker, and we can discuss the full PR later.

File: applications/compare_cumulative_psf.py [hunk 1]
```diff
@@ -35,14 +35,13 @@ def getData(**kwargs):
...
     sourceDistance = 12 * u.km
-    site = 'south'
     version = 'prod4'
     label = 'lst_integral'
     zenithAngle = 20 * u.deg
     offAxisAngle = [0 * u.deg]
 
     tel = TelescopeModel(
-        telescopeName='north-lst-1',
+        telescopeType='north-lst-1',
```
Comment (RaulRPrado) [inline applications/compare_cumulative_psf.py:44]: Why did you change it to telescopeType? It shouldn't work now. I changed it to telescopeName every where.

Comment (orelgueta) [reply applications/compare_cumulative_psf.py:44]: A mistake that was propagated accidentally because I had to merge in order to test the Docker.
Sorry about that.

Comment (RaulRPrado) [reply applications/compare_cumulative_psf.py:44]: No problem. I think we should fix it directly in the master, just to prevent propagating it. Can you do that or I do?

Comment (orelgueta) [reply applications/compare_cumulative_psf.py:44]: I don't think I am allowed to push directly to the master, at least it was rejected last time I tried...

Comment (RaulRPrado) [reply applications/compare_cumulative_psf.py:44]: That is weird. I'm allowed.
I'll fix it now and check what is going on in your case.

Comment (RaulRPrado) [reply applications/compare_cumulative_psf.py:44]: I learned how to set this kind of restriction but there is none set for this repository. Maybe you could try again someday jut to be sure.

Comment (orelgueta) [reply applications/compare_cumulative_psf.py:44]: OK, weird, maybe I misunderstood a previous error. Will try again in the future.
